homestar-runtime/src/settings.rs

```diff
@@ -22,6 +24,55 @@ pub struct Monitoring {
     process_collector_interval: u64,
 }
 
+#[derive(Default, Clone, Debug, Deserialize)]
+/// Configure how the Network keypair is generated or using an existing one
+pub(crate) enum PubkeyConfig {
+    #[default]
+    #[serde(rename = "random")]
+    Random,
```

Comment:
To your point in chat @mriise, I'd be ok removing `generatefromseed` for just using a seed for all homestar nodes internally that is our default. What do you think?  Other systems, mainly blockchain-related stuff using libp2p, always gave the option, but I'm not stuck to it.

Reply:
fair enough to have it though, its a bit silly- but since we give the option of random, specifying a different seed is important enough.